dhis-2/dhis-api/src/main/java/org/hisp/dhis/icon/IconResponse.java

```diff
@@ -56,10 +56,14 @@ public class IconResponse {
   @JsonProperty("href")
   private String reference;
 
-  public IconResponse(String key, String description, String[] keywords, String reference) {
+  @JsonProperty private boolean custom;
+
+  public IconResponse(
```

Comment:
We should expose `created` and `lastUpdated` as well, shouldn't we?

Reply:
yes